wpilibcExamples/src/main/cpp/examples/MecanumControllerCommand/cpp/RobotContainer.cpp

```diff
@@ -65,48 +66,50 @@ frc2::Command* RobotContainer::GetAutonomousCommand() {
       // Pass the config
       config);
 
-  frc2::MecanumControllerCommand mecanumControllerCommand(
```

Comment:
Instead of constructing the `MecanumControllerCommand` inline, construct it here and move into the sequence -- as before. Your choice whether to `ToPtr` it here at construction or when moving it into the group.